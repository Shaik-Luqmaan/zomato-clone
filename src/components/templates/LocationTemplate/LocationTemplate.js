import React from 'react'
import {Grid} from '@material-ui/core';
import Typography from'../../atoms/Typography/Typography';
import { makeStyles } from "@material-ui/core/styles";
import Locations from '../../organism/Location/LocationCards'

const useStyles = makeStyles({
  root:{
    padding: "40px 30px 0 0",
  },
  text: {
      padding: "10px 50px 0 80px",
      color: "rgb(54, 54, 54)",
      fontSize: "3.6rem",
      lineHeight: 1.2,
      fontWeight: 400,
  },
});

function LocationTemplate(props) {
  const classes = useStyles();
  return (
    <Grid container direction="column" className={classes.root} alignItems="center">
    <Grid item className={classes.text}>
      <Typography variant="h4">Popular localities in and around Hyderabad</Typography>
    </Grid>
    <Grid item>
        <Locations />
    </Grid>
  </Grid>
  )
}

export default LocationTemplate
