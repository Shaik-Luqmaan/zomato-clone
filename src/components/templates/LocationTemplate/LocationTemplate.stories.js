import React from 'react';
import LocationTemplate from './LocationTemplate'

export default {
    title: 'templates/Location',
    component: LocationTemplate
}

const Template = (...args) => (
    <LocationTemplate {...args} />
);

export const Primary = Template.bind({});