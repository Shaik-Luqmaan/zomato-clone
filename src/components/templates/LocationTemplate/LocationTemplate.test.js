import React from "react";
import ReactDOM from "react-dom";
import { BrowserRouter } from "react-router-dom";
import "@testing-library/jest-dom";
import LocationTemplate from "./LocationTemplate";
import renderer from "react-test-renderer";

it("matches snapshot", () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <LocationTemplate />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders LocationTemplate without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(
    <BrowserRouter>
      <LocationTemplate />
    </BrowserRouter>,
    div
  );
});