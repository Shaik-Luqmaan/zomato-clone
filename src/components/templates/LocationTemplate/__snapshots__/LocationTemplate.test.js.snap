// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="MuiGrid-root makeStyles-root-1 MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-items-xs-center"
>
  <div
    className="MuiGrid-root makeStyles-text-2 MuiGrid-item"
  >
    <h4
      className="MuiTypography-root MuiTypography-h4"
    >
      Popular localities in and around Hyderabad
    </h4>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item"
  >
    <div
      className="MuiGrid-root makeStyles-root-3 MuiGrid-container MuiGrid-spacing-xs-2"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <div
          className="MuiGrid-root makeStyles-root-4 MuiGrid-container MuiGrid-align-items-xs-center"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
          >
            <h6
              className="MuiTypography-root MuiTypography-h6"
              data-testid="location"
            >
              Tolichowki
               (
              540
              )
            </h6>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <div
          className="MuiGrid-root makeStyles-root-4 MuiGrid-container MuiGrid-align-items-xs-center"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
          >
            <h6
              className="MuiTypography-root MuiTypography-h6"
              data-testid="location"
            >
              Mehdipatnam
               (
              451
              )
            </h6>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <div
          className="MuiGrid-root makeStyles-root-4 MuiGrid-container MuiGrid-align-items-xs-center"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
          >
            <h6
              className="MuiTypography-root MuiTypography-h6"
              data-testid="location"
            >
              Asif Nagar
               (
              187
              )
            </h6>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <div
          className="MuiGrid-root makeStyles-root-4 MuiGrid-container MuiGrid-align-items-xs-center"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
          >
            <h6
              className="MuiTypography-root MuiTypography-h6"
              data-testid="location"
            >
              Lakdikapool
               (
              122
              )
            </h6>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <div
          className="MuiGrid-root makeStyles-root-4 MuiGrid-container MuiGrid-align-items-xs-center"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
          >
            <h6
              className="MuiTypography-root MuiTypography-h6"
              data-testid="location"
            >
              Nampally
               (
              48
              )
            </h6>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <div
          className="MuiGrid-root makeStyles-root-4 MuiGrid-container MuiGrid-align-items-xs-center"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
          >
            <h6
              className="MuiTypography-root MuiTypography-h6"
              data-testid="location"
            >
              Jubilee Hills
               (
              363
              )
            </h6>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <div
          className="MuiGrid-root makeStyles-root-4 MuiGrid-container MuiGrid-align-items-xs-center"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
          >
            <h6
              className="MuiTypography-root MuiTypography-h6"
              data-testid="location"
            >
              Gachibowli
               (
              584
              )
            </h6>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <div
          className="MuiGrid-root makeStyles-root-4 MuiGrid-container MuiGrid-align-items-xs-center"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
          >
            <h6
              className="MuiTypography-root MuiTypography-h6"
              data-testid="location"
            >
              Hitech City
               (
              264
              )
            </h6>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <div
          className="MuiGrid-root makeStyles-root-4 MuiGrid-container MuiGrid-align-items-xs-center"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
          >
            <h6
              className="MuiTypography-root MuiTypography-h6"
              data-testid="location"
            >
              Madhapur
               (
              93
              )
            </h6>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <div
          className="MuiGrid-root makeStyles-root-4 MuiGrid-container MuiGrid-align-items-xs-center"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
          >
            <h6
              className="MuiTypography-root MuiTypography-h6"
              data-testid="location"
            >
              Banjara Hills
               (
              363
              )
            </h6>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <div
          className="MuiGrid-root makeStyles-root-4 MuiGrid-container MuiGrid-align-items-xs-center"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
          >
            <h6
              className="MuiTypography-root MuiTypography-h6"
              data-testid="location"
            >
              Suncity
               (
              84
              )
            </h6>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <div
          className="MuiGrid-root makeStyles-root-4 MuiGrid-container MuiGrid-align-items-xs-center"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
          >
            <h6
              className="MuiTypography-root MuiTypography-h6"
              data-testid="location"
            >
              Santosh Nagar
               (
              264
              )
            </h6>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <div
          className="MuiGrid-root makeStyles-root-4 MuiGrid-container MuiGrid-align-items-xs-center"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
          >
            <h6
              className="MuiTypography-root MuiTypography-h6"
              data-testid="location"
            >
              Madhapur
               (
              93
              )
            </h6>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <div
          className="MuiGrid-root makeStyles-root-4 MuiGrid-container MuiGrid-align-items-xs-center"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
          >
            <h6
              className="MuiTypography-root MuiTypography-h6"
              data-testid="location"
            >
              Kukatpally
               (
              1153
              )
            </h6>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <div
          className="MuiGrid-root makeStyles-root-4 MuiGrid-container MuiGrid-align-items-xs-center"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
          >
            <h6
              className="MuiTypography-root MuiTypography-h6"
              data-testid="location"
            >
              Old City
               (
              166
              )
            </h6>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
              />
            </svg>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
